fluentd/run.sh

```diff
@@ -2,6 +2,7 @@
 
 export MERGE_JSON_LOG=${MERGE_JSON_LOG:-true}
 CFG_DIR=/etc/fluent/configs.d
+ENABLE_PROMETHEUS_ENDPOINT=${ENABLE_PROMETHEUS_ENDPOINT:-"true"}
```

Comment:
should we default this to false? Since it potentially will increase the memory footprint of Fluentd, this way a team can properly capacity plan (e.g. ops)

Reply:
My opinion is we should enable it and modify https://bugzilla.redhat.com/show_bug.cgi?id=1590920 to increase the memory as it seems this will resolve the memory issue.